src/main/java/redis/clients/jedis/JedisClusterCommand.java

```diff
@@ -16,12 +18,25 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(JedisClusterCommand.class);
 
+  private static final Duration DEFAULT_MAX_TOTAL_RETRIES_DURATION = Duration.ofMillis(
+      BinaryJedisCluster.DEFAULT_TIMEOUT * BinaryJedisCluster.DEFAULT_MAX_ATTEMPTS);
+
   private final JedisClusterConnectionHandler connectionHandler;
   private final int maxAttempts;
+  private final Duration maxTotalRetriesDuration;
 
   public JedisClusterCommand(JedisClusterConnectionHandler connectionHandler, int maxAttempts) {
+    this(connectionHandler, maxAttempts, DEFAULT_MAX_TOTAL_RETRIES_DURATION);
```

Comment:
```suggestion
    this(connectionHandler, maxAttempts, Duration.ofMillis((long) BinaryJedisCluster.DEFAULT_TIMEOUT * maxAttempts));
```

Reply:
The change here is invalid under the current situation, `maxTotalRetriesDuration` will be modified to `soTimeout` again in `BinaryJedisCluster.java`. Because user code always needs `new JedisCluster(xxx)`.

I think there are three solutions:

1. Modify the `BinaryJedisCluster` constructor
```
this.maxTotalRetriesDuration = Duration.ofMillis(soTimeout * maxAttempts);
```

2. Add a constructor in `JedisCluster.java`, so we can directly specify `maxTotalRetriesDuration`

```
public JedisCluster(Set<HostAndPort> jedisClusterNode, int connectionTimeout, int soTimeout,
      int infiniteSoTimeout, int maxAttempts, String user, String password, String clientName,
      final GenericObjectPoolConfig<Jedis> poolConfig, boolean ssl,
      SSLSocketFactory sslSocketFactory, SSLParameters sslParameters,
      HostnameVerifier hostnameVerifier, JedisClusterHostAndPortMap hostAndPortMap, Duration maxTotalRetriesDuration) {
    super(jedisClusterNode, connectionTimeout, soTimeout, infiniteSoTimeout, maxAttempts, user,
        password, clientName, poolConfig, ssl, sslSocketFactory, sslParameters, hostnameVerifier,
        hostAndPortMap, maxTotalRetriesDuration);
  }
```

3. Set `maxTotalRetriesDuration` as a separate configuration (discussed, not working for this PR)